galleries/examples/style_sheets/style_sheets_reference.py

```diff
@@ -5,8 +5,18 @@
 
 This script demonstrates the different available style sheets on a
 common set of example plots: scatter plot, image, bar graph, patches,
-line plot and histogram,
+line plot and histogram.
 
+Any of these style sheets can be imported (i.e. activated) by its name.
+For example for the ggplot style:
+
+>>> plt.style.use('ggplot')
+
+The names of the available style sheets can be found
+in the list `matplotlib.style.available`
+(they are also printed in the corner of each plot below).
+
+See more details in :ref:`Customizing Matplotlib using style sheets<customizing-with-style-sheets>`.
```

Comment:
```suggestion
See more details in
:ref:`Customizing Matplotlib using style sheets<customizing-with-style-sheets>`.
```
should work (in the sense that there will not be a line break there in the rendered page).

Reply:
You can break anywhere in the backticks, so just break normally on spaces.